src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,50 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
```

Comment:
We've tended to put these in the constants files in the past.

Though honestly this is probably way nicer.

Not sure about the path though; I don't think it is really needed/helpful because it's easy to grep for when debugging and it just adds size to the app/console output/etc.